components/Dialogs/src/TypographyThemer/MDCAlertTypographyThemer.h

```diff
@@ -14,11 +14,11 @@
  limitations under the License.
  */
 
-#import <UIKit/UIKit.h>
-
 #import "MaterialDialogs.h"
 #import "MaterialTypographyScheme.h"
 
+#import <Foundation/Foundation.h>
```

Comment:
Why the UIKit.h/Foundation.h swap?

Reply:
This file didn't need UIKit.h and I'm attempting to avoid having hidden import dependencies.